src/ngraph/runtime/plaidml/plaidml_ops_group_convolution.hpp

```diff
@@ -0,0 +1,60 @@
+//*****************************************************************************
+// Copyright 2017-2019 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#pragma once
+
+#include "ngraph/axis_vector.hpp"
+#include "ngraph/op/convolution.hpp"
+#include "ngraph/op/fused/group_conv.hpp"
+#include "ngraph/op/op.hpp"
+
+namespace ngraph
+{
+    namespace runtime
+    {
+        namespace plaidml
+        {
+            namespace op
+            {
+                class GroupConvolution;
```

Comment:
How is this class being used? I don't see anything that actually instantiates it.

Reply:
We don't have anything that instantiates it - this is just an artifact of the `Convolution` op (which I used as a template, and which actually has other passes that instantiate it). If we don't think anything will instantiate GroupConvolution, I can get rid of it.

Reply:
Gotcha- might be best to get rid of it in that case, but up to you.

Reply:
@tzerrell what do you think -- should we be keeping this or getting rid of it?

Reply:
I don't particularly see a need for it, but perhaps @earhart would know differently?

Reply:
We shouldn't need the .hpp file for this at all.  (Convolution has one since there's a pass that folds transposes into convolutions.)

Reply:
The consensus is that I'll remove this class. Thanks for the input!